sunflower-land/sunflower-land
# [CHORE] Add animation domains

# Description

Adds domains for the animations service

## How to Test

1. Ensure `VITE_ANIMATION_URL` is set in your `.env`.
2. Load up the game locally. Test the following two example URLs:

http://localhost:3000/#/world/examples_rpg
http://localhost:3000/#/world/examples_animations

## Type of change

Please delete options that are not relevant.


# How Has This Been Tested?

localhost

# Checklist:
